hashes/src/sha256t.rs

```diff
@@ -211,7 +176,7 @@ macro_rules! sha256t_tag {
 ///
 /// [`hash_newtype`]: crate::hash_newtype
 #[macro_export]
-#[deprecated(since = "TBD", note = "use `sha256_tag!` combined with `hash_newtype!` instead")]
+#[deprecated(since = "0.15.0", note = "use `sha256_tag!` combined with `hash_newtype!` instead")]
```

Comment:
Add `/// # #[allow(deprecated)] {` to the example code in `hashes/src/sha256t.rs:154` to remove the lint warning.

Reply:
Thanks man, I don't know why it passed CI. I created #3529